frameworks/FSharp/giraffe/src/App/App.fsproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <EnableDefaultContentItems>false</EnableDefaultContentItems>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="6.0.0" />
-    <PackageReference Include="Dapper" Version="2.0.90" />
-    <PackageReference Include="Giraffe" Version="5.0.0" />
-    <PackageReference Include="Npgsql" Version="5.0.7" />
+    <PackageReference Update="FSharp.Core" Version="6.0.1" />
+    <PackageReference Include="Dapper" Version="2.0.123" />
+    <PackageReference Include="Giraffe" Version="6.0.0-alpha-1" />
+    <PackageReference Include="Npgsql" Version="5.0.10" />

```

Comment:
We can go to version `6.0.0` now for `Npgsql`

Reply:
thanks